README.md

```diff
@@ -55,7 +57,12 @@ You can just pass a `-print` argument and `govvv` will just print the
 
 Still don’t want to wrap the `go` tool? Well, try `-flags` to retrieve the LDFLAGS govvv prepares:
 
-    $ go build -ldflags="$(govvv -flags)"
+```
+$ go build -ldflags="$(govvv -flags)"
+
+# or specifying 'version' subpackage
```

Comment:
this will confuse most people. Let's not change this area either. Saying `-pkg` exists at one location is enough.